Kickstarter-iOS/Views/Cells/SettingsPrivacyRequestDataCell.swift

```diff
@@ -0,0 +1,108 @@
+import KsApi
+import Library
+import Prelude
+import Prelude_UIKit
+import ReactiveSwift
+import UIKit
+
+public protocol SettingsRequestDataCellDelegate: class {
+  func shouldPresentRequestDataPrompt()
```

Comment:
Wondering if the naming here can be improved, how does something like this sound:

```swift
func settingsRequestDataCellDidPrompt(_ cell: SettingsPrivacyRequestDataCell)
func settingsRequestDataCell(_ cell: SettingsPrivacyRequestDataCell, requestedDataWith url: URL)
```

Reply:
Great suggestion Justin! I changed the name slightly to ` settingsRequestDataCellDidPresentPrompt(_ cell: SettingsPrivacyRequestDataCell)` adding the "present" made it a little more clear to me

Reply:
Great, makes sense!